demo/full/scripts/components/BufferContentGraph.tsx

```diff
@@ -148,11 +148,9 @@ export default function BufferContentGraph({
   const paintSegment = React.useCallback(
     (scaledSegment: IScaledBufferedData, canvasCtx: CanvasRenderingContext2D): void => {
       const representation = scaledSegment.bufferedInfos.infos.representation;
-      let indexOfRepr = representationsIdEncountered.current.indexOf(
-        representation.uniqueId,
-      );
+      let indexOfRepr = representationsIdEncountered.current.indexOf(representation.id);
       if (indexOfRepr < 0) {
-        representationsIdEncountered.current.push(representation.uniqueId);
+        representationsIdEncountered.current.push(representation.id);
```

Comment:
Why not using `uniqueId`?
There's a small risk of two Representations from two different Adaptation having the same `id`

Reply:
updated!